spec/requests/carto/api/visualizations_controller_spec.rb

```diff
@@ -1578,6 +1579,11 @@ def first_data_layer_from_response(response)
     end
 
     it 'orders remotes by size with external sources size' do
+      pending("Bloomberg has a different behavior between index and load_common_data")
```

Comment:
Could we write a set of tests to check the new behaviour? We could eve prepend them in a describe that indicates their Bloomberg's logic